### ISSUE TITLE
browse to specific slice when opening perfetto ui website by self-hosted web page 

### ISSUE THREAD

**Issue by wangjiyang** [google/perfetto#897]
I would like to use deep linking to the Perfetto UI, then open the trace at a specific slice ID and automatically scroll to that slice ID in the graphical interface, as shown below:
![3850347825469ccfe96a3edebb489c9c](https://github.com/user-attachments/assets/6862e4c9-d697-41ef-b820-602cecc09f88)

To achieve this goal, I used the following code:

	setTimeout(() => {
	    win.postMessage(
	        {
	            perfetto: {
	                timeStart: 385945454908325 / 1e9,
	                timeEnd: 385947619633249 / 1e9,
	                viewPercentage: 0.5,
	            }
	        },
	        ORIGIN
	    );
	    console.log('Message sent after delay: 15s');
	}, 15000);
However, I found that it can only horizontally scroll to the timestamp near the stutter, but it does not vertically scroll to this slice ID. Please help me analyze how I can achieve the expected effect.

**Comment by LalitMaganti** [comment]
Please read https://perfetto.dev/docs/visualization/deep-linking-to-perfetto-ui#opening-the-trace-at-a-specific-event-or-time

TLDR if you want to select a given slice you need to specify "ts" and "dur" which generally uniquely identifies a slice.
